File: Cap 06/Topico 6.2/Media Geral V2/medverv2.c
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 6 - Topico 6.2
 * 
 * Programa ..: MEDIA GERAL - VERSAO 2
 */

#include <stdio.h>

int I;
float MD[8];
float SOMA = 0, MEDIA;
int main(void) {
  printf("MEDIA GERAL - V2\n\n");
  SOMA = 0;
  for (I = 0; I <= 7; I++) {
    printf("Entre a nota %d: ", I + 1);
    scanf("%f", &MD[I]);
    SOMA = SOMA + MD[I];
  }
  MEDIA = SOMA / 8;
  printf("Resultado da media = %5.1f\n", MEDIA);
  return 0;
}

File: Cap 06/Topico 6.2/Media Geral V2/medverv2.scala
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 6 - Topico 6.2
 * 
 * Programa ..: MEDIA GERAL - VERSAO 2
 * 
 */

object Main {
  def main(args: Array[String]): Unit = {
    println("MEDIA GERAL - V2\n")
    var MD = new Array[Double](8)
    var SOMA = 0.0
    var MEDIA = 0.0
    for (I <- 0 to 7 by 1) {
      print(s"Entre a nota ${I + 1}: ")
      MD(I) = scala.io.StdIn.readDouble()
      SOMA = SOMA + MD(I)
    }
    MEDIA = SOMA / 8
    printf("Resultado da media = %5.1f\n", MEDIA)
  }
}

